README.md

```diff
@@ -97,6 +97,7 @@ Open the Playground from within the `.xcworkspace` in order for it to work.
   - [ArrayExtension](#arrayextension)
   - [DictionaryExtension](#dictionaryextension)
   - [DispatchTimeIntervalExtension](#dispatchtimeintervalextension)
+  - [CollectionExtension](#collectionextension)
```

Comment:
I think, from the order perspective, it makes more sense that `Collection` appears somewhere around array. I think I'd place it before `Array` even.